### ISSUE TITLE
Custom configuration is not used when using command-line formatting

### ISSUE THREAD

**Issue by GoldSubmarine** [cortinico/ktfmt-gradle#34]
## 🐛 Describe the bug

Custom configuration is not used when using command-line formatting

## ⚠️ Current behavior

Use the `./gradlew ktfmtFormat` command to format the code

![image](https://user-images.githubusercontent.com/22909744/121156603-6a024b80-c87b-11eb-98e9-a8e204048357.png)

Use the `./gradlew ktfmtPrecommit --include-only="......."` command to format the code

![image](https://user-images.githubusercontent.com/22909744/121156557-5f47b680-c87b-11eb-9170-8b0fe6b2e9d9.png)

My configuration file is as follows

```kotlin
ktfmt {
    kotlinLangStyle()
    maxWidth.set(120)
}
```

## 📱 Tech info
 - Device: dell
 - OS: Win10
 - Library/App version: 0.6.0

**Comment by cortinico** [comment]
Can you paste the code where you define the `ktfmtPrecommit` task?

**Comment by GoldSubmarine** [comment]
```kotlin
tasks.register<KtfmtFormatTask>("ktfmtPrecommit") {
    source = project.fileTree(rootDir)
    include("**/*.kt")
}
```

**Comment by cortinico** [comment]
Thanks. So I need to fix this internally. Ideally the task should check if there is a `ktfmt{}` extension registered before falling back to default values.

**Comment by cortinico** [comment]
Sorry this took a bit longer, but #36 should address your scenario

**Comment by GoldSubmarine** [comment]
Hi, can you merge the PR, and publish a new version?

**Comment by cortinico** [comment]
> Hi, can you merge the PR, and publish a new version?

Yup I'll do it in the next days 👍

**Comment by cortinico** [comment]
Shipped in version `0.7.0` of the plugin
